src/v/utils/tests/retry_chain_node_test.cc

```diff
@@ -212,6 +212,17 @@ SEASTAR_THREAD_TEST_CASE(check_tracing) {
             != std::string_view::npos;
     BOOST_REQUIRE(found);
 
+    std::vector<ss::sstring> actual;
```

Comment:
Looks like the meat of this commit was squashed into the first one. Squash then the test too otherwise the commit message is confusing and irrelevant.